src/components/Column/Column.scss

```diff
@@ -49,6 +49,7 @@
 }
 
 .column__header {
+  z-index: 1;
```

Comment:
lgtm, just one small thing, could you use one of the variables provided in `style.scss` instead of a hardcoded value? Like `$base-z-index`.